components/style_traits/values/mod.rs

```diff
@@ -4,6 +4,9 @@
 
 //! Helper types and traits for the handling of CSS values.
 
+#[allow(missing_docs)]
```

Comment:
Please add docs instead, or add the annotations to the particular things that don't have docs. Thought preferably add them! :)